content/community/governance/geb/index.md

```diff
@@ -1,12 +1,17 @@
-# Galaxy Executive Board
+---
+title: Galaxy Executive Board (GEB)
+autotoc: false
+---
 
 **Keywords:** Strategic emphasis, long term planning and direction
 
-The *[Galaxy Exec](/community/exec/)* is a group of principal investigators from around the world, that is responsible for [coordinating the direction](/community/governance/) of the [Galaxy Project](/), a mature, successful research software infrastructure project, with a large and diverse global audience.
+The [GEB](/community/governance/geb/) is a group of principal investigators from around the world, that is responsible for [coordinating the direction](/community/governance/) of the [Galaxy Project](/), a mature, successful research software infrastructure project, with a large and diverse global audience.
 
-The Galaxy Exec was formed at the _Galaxy Directions Summit_ in 2019, which gathered together ~20 PIs from around the world. Initially, all Exec members run large grants that have Galaxy as a core part of their aims.
+The Galaxy Executive Board was formed at the _Galaxy Directions Summit_ in 2019, which gathered together ~20 PIs from around the world. Initially,
+all Executive members run large grants that have Galaxy as a core part of their aims.
 
 ## Mission Statement
```

Comment:
Can we make this a 'hook'? It's duplicated on the governance index page and if we just had hooks for each thing in this folder it would be easier to update.

Reply:
Oh is it the same? I guess the mission statement of the GEB can (should?) be different from the overall project?

Reply:
I think it's more that the definitions of each of the groups are repeated on the governance index and then again on their main page (and possibly elsewhere) so we can head off inconsistencies by making a hook/snippet.
Not the mission statement, that for the GEB can be on the GEB page (in the future!), so just leave the excellent current overall Galaxy mission statement.
This comment was on defining the GEB

Reply:
That is a good idea, but I do not know how to do this in this current version of the Hub :(